Solinea/goldstone-server
# Saved search client

also includes https://github.com/Solinea/goldstone-compliance/pull/41

to test:
* check out branch
* navigate to http://localhost:8000/#reports/logbrowser and click the gear to get to the settings page
* create/update/delete searches to test
* navigate back to http://localhost:8000/#reports/logbrowser to see if your searches were added to the drop down.


sets up ui and editing page/forms for saved searches.

quick fixes:
placeholder in resource list on topology page.
axis adjustments for resources usage charts on discover page.
